.pre-commit-config.yaml

```diff
@@ -1,3 +1,7 @@
+ci:
+  autoupdate_commit_msg: "chore: update pre-commit hooks"
+  autofix_commit_msg: "style: pre-commit fixes"
```

Comment:
@ioanaif, this is how you can change all the auto PR names instead of doing it one at a time. :)

Reply:
Thanks! I'm the one who started changing them one at a time because I was too impatient to look into how to do it once and for all, and it became a pattern for us.